queries.sql

```diff
@@ -1,3 +1,170 @@
 /*Queries that provide answers to the questions from all projects.*/
+/*
+ connect to the database if you created already otherwise go to the respective file
+ 
+ follow this in psql shell
+ 
+ Server [localhost]:
+ Database [postgres]: vet_clinic
+ Port [5432]:
+ Username [postgres]:
+ Password for user postgres:
+ psql (15.2)
+ WARNING: Console code page (437) differs from Windows code page (1252)
+ 8-bit characters might not work correctly. See psql reference
+ page "Notes for Windows users" for details.
+ Type "help" for help.
+ 
+ vet_clinic=# SELECT * from animals;
+ id |  name   | date_of_birth | escape_attempts | neutered | weight_kg
+ ----+---------+---------------+-----------------+----------+-----------
+ 1 | Agumon  | 2020-02-03    |               0 | t        |     10.23
+ 2 | Gabumon | 2018-11-15    |               2 | t        |         8
+ 3 | Pikachu | 2021-01-07    |               1 | f        |     15.04
+ 4 | Devimon | 2017-05-12    |               5 | t        |        11
+ (4 rows)
+ 
+ */
+-- Find all animals whose name ends in "mon".
+SELECT
+  name
+FROM
+  animals
+WHERE
+  name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+/* output expected
+ name
+ ---------
+ Agumon
+ Gabumon
+ Devimon
+ (3 rows)
+ */
+--  List the name of all animals born between 2016 and 2019.
+SELECT
+  name
+FROM
+  animals
+WHERE
+  date_of_birth BETWEEN '2016-01-01'
+  AND '2019-12-31';
+
+/* output expected
+ name
+ ---------
+ Gabumon
+ Devimon
+ (2 rows)
+ */
+--  List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT
+  name
+FROM
+  animals
+WHERE
+  escape_attempts < '3';
```

Comment:
- [ Required ] I think the project requirements state that this query should return all animals that are `neutered` and have less than `3` escape attempts. Currently, this query returns animals both neutered and not neutered that have less than `3` escape attempts. Kindly rectify this.

Reply:
- sure i think i missed that one 😁😁😁😂

Reply:
# Updated SS

![image](https://user-images.githubusercontent.com/116451144/230286593-4afa07e2-0bca-468e-90c9-19d4c1643570.png)